.github/workflows/osrm-backend.yml

```diff
@@ -0,0 +1,45 @@
+name: osrm-backend
+
+on: [workflow_dispatch]
+
+jobs:
+  build_wheels:
+    name: osrm-backend, ${{ matrix.platform }}
+    runs-on: ${{ matrix.platform }}-${{ matrix.version }}
+    strategy:
+      fail-fast: true
+      matrix:
+        platform: [macos]
+        version: [latest]
+    env:
+      GITHUB_TOKEN: ${{ github.token }}
+      ARTIFACT_WORKFLOW: osrm-backend.yml
+
+    steps:
+    - name: Checkout osrm-backend
+      uses: actions/checkout@v3
+      with:
+        repository: Project-OSRM/osrm-backend
+        submodules: true
+        fetch-depth: 1
+        path: osrm-backend
+
+    - name: Build osrm-backend
+      run: |
+        brew install lua tbb
+        brew install boost@1.76 && brew link boost@1.76
```

Comment:
is there a reason why we pin boost to 1.76?

Reply:
I think so, AFAIR there is a bug in the latest boost which doesn't allow OSRM to build.

Reply:
Can confirm. There was a duplicate symbols issue that I encountered previously on latest, and brew didn't have 1.79 - hence the 1.76 pin.